util/query.py

```diff
@@ -336,12 +342,47 @@ def _filterOutPurgatory(nftvols: dict, chainID: int) -> dict:
       filtered_nftvols: list of [basetoken_addr][nft_addr]:vol_amt
     """
     bad_dids = _didsInPurgatory()
-    filtered_pools = {}
+    filtered_nfts: Dict[str, Dict[str, float]] = {}
     for basetoken_addr in nftvols:
         for nft_addr in nftvols[basetoken_addr]:
             if oceanutil.calcDID(nft_addr, chainID) not in bad_dids:
-                filtered_pools[basetoken_addr] = nftvols[basetoken_addr]
-    return filtered_pools
+                filtered_nfts[basetoken_addr] = nftvols[basetoken_addr]
+    return filtered_nfts
+
+
+@enforce_types
+def _filterOutNonMarketAssets(nftvols: dict, chainID: int) -> dict:
```

Comment:
I'd re-label from "_filterOutNonMarketAssets" to "_filterToAquariusAssets". Why:
- "non-market" is vague. Aquarius is precise
- replace the double-negative ("out" and "non-market") with a positive ("just aquarius")